### ISSUE TITLE
Upgrade WireMock

### ISSUE THREAD

**Issue by otrosien** [ePages-de/restdocs-wiremock#6]
Upgrade to 2.1.0-beta

**Comment by otrosien** [comment]
Blocker: Seems like 2.1.0 removed the JsonFileMappingsLoader. Need to clarify with @tomakehurst what to use instead.

**Comment by otrosien** [comment]
Ok, found commit in wiremock (https://github.com/tomakehurst/wiremock/commit/a093d20ddc15cb6d1f9871d06d2878c7839a7058), so supposedly I need to use `JsonFileMappingsSource` directly.

**Comment by tomakehurst** [comment]
Yes, that's the one you probably want to use going forward.

**Comment by otrosien** [comment]
Thanks! Closed via f3b2293

**Comment by otrosien** [comment]
Unfortunately this causes a NPE.

**Comment by otrosien** [comment]
Problem is line 85 of JsonFileMappingsSource. getUri().getPath() returns null for a resource that is not in a resource folder, but coming from inside a jar file. I'll open a ticket in the wiremock project.

```java
mappingFile.getUri().getPath().replace(mappingsFileSource.getUri().getPath(), "");
```

Stacktrace:
```
java.lang.NullPointerException
	at com.github.tomakehurst.wiremock.standalone.JsonFileMappingsSource.getFileName(JsonFileMappingsSource.java:85)
	at com.github.tomakehurst.wiremock.standalone.JsonFileMappingsSource.loadMappingsInto(JsonFileMappingsSource.java:72)
	at com.github.tomakehurst.wiremock.core.WireMockApp.loadMappingsUsing(WireMockApp.java:87)
	at com.github.tomakehurst.wiremock.core.WireMockApp.loadDefaultMappings(WireMockApp.java:83)
..
```

**Comment by otrosien** [comment]
Closed via e4af636
